docs/source/changelog.rst

```diff
@@ -1,4 +0,0 @@
-Changelog
-=========
-
-This is a stub file.
```

Comment:
I just realized that the changelog got deleted. I'm strongly in favor of keeping it somewhere in the documentation, as a separate chapter. Can we put it back please? (I know this is just a stub file, but still).

Reply:
Yes, it will come back! Lets discuss this in #290